cpp/include/cugraph/edge_property.hpp

```diff
@@ -113,6 +117,13 @@ class edge_property_t {
   {
   }
 
+  // nvcc 12.5 sometimes deduces this as a host device function, just defining it fixes that
+  ~edge_property_t() {}
+  edge_property_t(const edge_property_t&)            = default;
```

Comment:
Copy constructor & operator should be deleted. This class is a wrapper (+ alpha) for rmm::device_uvector.

Reply:
@miscco - we should correct this before merging this change.

Reply:
@ChuckHastings @seunghwak Changed to delete